claat/serve.go

```diff
@@ -72,214 +60,232 @@ func cmdServe() {
 	fatalf("claat serve: %v", <-ch)
 }
 
-// downloadFile will download a url to a local file. It's efficient because it will
-// write as it downloads and not load the whole file into memory.
-func downloadFile(filepath string, url string) error {
-	if !warned {
-		fmt.Println("Fetching dependencies...")
-		warned = true
-	}
-	out, err := os.Create(filepath)
-	if err != nil {
-		return err
-	}
-	defer out.Close()
-	resp, err := http.Get(url)
-	if err != nil {
+func writeFile(name string, content []byte) error {
```

Comment:
Would it be more idiomatic to pass a Reader interface, rather than a byte string?